modules/charon-core/src/main/java/org/wso2/charon/core/protocol/endpoints/UserResourceEndpoint.java

```diff
@@ -181,7 +181,7 @@ public SCIMResponse create(String scimObjectString, String inputFormat, String o
             //if the submitted format not supported, encode exception and set it in the response.
             return AbstractResourceEndpoint.encodeSCIMException(encoder, e);
         } catch (CharonException e) {
-            logger.error("Internal server error while creating new resource.", e);
+            logger.error("Internal server error while creating new resource: " + e.getDescription(), e);
```

Comment:
Shouldn't all the information in "e" be printed with the log as we pass "e"? is this printing any additional information?

Reply:
Here e.getDescription gives a brief explanation of the exception. For example, if we get an exception when creating user as in [1] then the brief explanation of the exception will be pass through [2] & [3], So e.getmessage will be null and at the same time e.getDescription will give the explanation.

[1] https://github.com/wso2-extensions/identity-inbound-provisioning-scim/blob/master/components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/impl/SCIMUserManager.java#L130

[2] https://github.com/wso2/charon/blob/11991d5c970d23fc9266813ed5edd84fa0262524/modules/charon-core/src/main/java/org/wso2/charon3/core/exceptions/CharonException.java#L40

[3] https://github.com/wso2/charon/blob/2.x.x/modules/charon-core/src/main/java/org/wso2/charon/core/exceptions/AbstractCharonException.java#L50